src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+= Listing in the App Store
+:page-slug: app-store
+:page-description: An introduction to the Koji App Store and how to list your Koji app.
+:includespath: ../_includes
+
+After you create, test, and move your app to production, you can share its production URL or list the app in the Koji App Store.
+
+The https://withkoji.com/apps[Koji App Store] is the marketplace for all Koji apps.
+When users edit their Koji profile, they can choose to add an app from the App Store.
```

Comment:
Let’s call them “creators” when we mean people who are going to configure and add apps to their profiles. I.e., "developers" code apps > "creators" configure apps > "users" use apps (or for generic cases).
```suggestion
When creators edit their Koji profile, they can choose to add an app from the App Store.
```

Reply:
Fixed.